Gemfile

```diff
@@ -60,6 +60,7 @@ gem 'roo'
 
 # Excel Export
 gem 'caxlsx'
```

Comment:
`caxlsx` is still used here: https://github.com/SaraAlert/SaraAlert/blob/master/lib/tasks/stats.rake#L568
`roo` is still used for imports since `fast_excel` only writes to excel files

Reply:
Thanks. Maybe we should look at swapping over stats.rake to use the fast_excel as well.